### PR TITLE
[Button] Removed example of stroked button

### PR DESCRIPTION
**Pull request by randallli**
Before:
simulator screen shot - iphone 8 plus - 2018-04-20 at 10 43 46

After:
simulator screen shot - iphone 8 plus - 2018-04-20 at 11 45 53

### CODE REVIEW

**Review by yarneo — CHANGES_REQUESTED** [review]
I am going to wait on approving this pending designer feedback

**Review by yarneo — APPROVED** [review]
Synced with design, this LGTM

**Comment by ianegordon** [comment]
All commits by Googlers @jverkoey or @randallli .

**Pull request merged**
